warehouse/query-core/src/main/java/datawave/query/jexl/visitors/ExpandMultiNormalizedTerms.java

```diff
@@ -274,6 +284,13 @@ protected JexlNode expandNodeForNormalizers(JexlNode node, Object data) {
                         nodeToReturn = JexlNodeFactory.buildUntypedNewLiteralNode(node, fieldName, literal);
                     }
                     
+                    // if we have an unnormalizable regex in the mix, then wrap with evaluation only
+                    // this is ok even if we have a mix of normalizations that worked vs not because
+                    // if one of a union is evaluation only, then they all must be (see executability
+                    // visitor for more understanding)
```

Comment:
Does this imply we need to push whole  unions into EvalOnly? `(A or B or (eval & C)) -> (eval & (A or B or C))`